### DISCUSSION TITLE
Which version of Beckn protocol core specification to implement

### DISCUSSION THREAD

**Discussion by tanweer555** [ONEST-Network/community#6] [Technical Queries] [answered]
Hi,
While working on integrating the BAP service APIs, I referred to the documentation available at https://github.com/beckn/beckn-sandbox/blob/main/USER_GUIDE.md to understand the sample APIs. The Beckn Protocol Core Specification version indicated in the documentation is v1.1.0. However, during the use case discussion session, there was a reference to using or implementing version v1.0 of the Beckn Protocol Core Specification. I'm seeking clarification regarding which version of the Beckn Protocol Core Specification should be implemented.

**Comment by faizmagic** [comment, marked as answer]
Hello @tanweer555 , ver 1.1.0 is just a minor upgrade to ver 1.0.0 of the protocol version, where there were some typo's which was corrected and in most cases we are referring ver 1.1.0 as ver 1.0.0, so when one is referring to ver 1.0.0 it is referring to ver 1.1.0

**Reply by tanweer555** [reply]
Thanks for the clarification @faizmagic
